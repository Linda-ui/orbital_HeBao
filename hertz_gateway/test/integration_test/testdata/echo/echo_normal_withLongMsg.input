{
  "msg": "Lorem ipsum dolor sit amet, consectetur adipiscing elituiofdsnfmdlauidansdfjusarhenwavjchx zmxc vkjfdhwjgndmVcuohqewLorem ipsum dolor sit amet, consectetur adipiscing elituiofdsnfmdlauidansdfjusarhenwavjchx zmxc vkjfdhwjgndmVcuohqew"
}