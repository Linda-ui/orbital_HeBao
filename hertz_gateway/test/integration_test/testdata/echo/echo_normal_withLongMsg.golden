{"err_code":0,"err_message":"success","msg":"Lorem ipsum dolor sit amet, consectetur adipiscing elituiofdsnfmdlauidansdfjusarhenwavjchx zmxc vkjfdhwjgndmVcuohqewLorem ipsum dolor sit amet, consectetur adipiscing elituiofdsnfmdlauidansdfjusarhenwavjchx zmxc vkjfdhwjgndmVcuohqew"}